styles/_uswds-theme-custom-styles.scss

```diff
@@ -202,6 +206,85 @@ HOME
   }
 }
 
+// ## DOMAIN SEARCH ---------------------------------------
```

Comment:
love that no where here is there a !important which I feel like people overuse to get around some of the css rules, thanks for adding all this styling!